### PR TITLE
Document the jurisprudence of the "fake news law"

### PR DESCRIPTION
**Pull request by BerylBogui**
Add jurisprudence.

### CODE REVIEW

File: _pages/encyclopedia/4-reaction/1-case-studies.md [hunk 1]
```diff
@@ -17,9 +17,7 @@ permalink: /encyclopedia/reaction/case-studies
 
 ### The French law against information manipulation
 
-In France, a law against information manipulation and the **intentional** spread of disinformation was officially adopted on November 20, 2018, nine months after its proposal.
-
-<!-- add decree publication date -->
+In France, a law against information manipulation and the **intentional** spread of disinformation was officially adopted on November 20, 2018, nine months after its proposal and the start of the accelereated procedure on 26 March 2018.
```
Comment (MattiSG) [inline _pages/encyclopedia/4-reaction/1-case-studies.md]: No, the decree publication date is the date on which the law is actually enacted, not the beginning of the procedure.

File: _pages/encyclopedia/4-reaction/1-case-studies.md [hunk 2]
```diff
@@ -30,6 +28,25 @@ During said times, the law compels platforms to be more transparent by, for inst
 Outside of election times, platforms ought to cooperate, as well as build and implement open measures to fight disinformation.
 The French Superior Council of the Audiovisual (CSA) has been given the authority to hinder or interrupt the broadcasting of foreign state-owned (or state-influenced) TV-services in cases where fundamental national interests are at stake.
 
+#### Jurisprudence
+
+On May 10, 2019 two French politicians (a European deputy and a senator) have asked for the **takedown of a Tweet** from the Minister of Interior claiming that a hospital in Paris was attacked during a demonstration on May 1 by a group Yellow Vests. However, it has been proven that these people were seeking refuge and had no intent to harm the hospital whatsoever.
```
Comment (MattiSG) [inline _pages/encyclopedia/4-reaction/1-case-studies.md]: - Unnecessary details on “politicians”.  “elected representatives” should be enough.
- Don't use bold.
- Add link to tweet and to proof.
- “only seeking refuge”, remove “had no intent…”

File: _pages/encyclopedia/4-reaction/1-case-studies.md [hunk 3]
```diff
@@ -30,6 +28,25 @@ During said times, the law compels platforms to be more transparent by, for inst
...
+
+Based on the 2018 law against information manipulation, they went to court on May 16 but  the **decision to clear both the Minister and Twitter** was given on May 17.
```
Comment (MattiSG) [inline _pages/encyclopedia/4-reaction/1-case-studies.md]: 10, not 16.

[Source](https://www.dalloz-actualite.fr/flash/loi-fake-news-premiere-application-du-refere)

Don't use bold.

File: _pages/encyclopedia/4-reaction/1-case-studies.md [hunk 4]
```diff
@@ -30,6 +28,25 @@ During said times, the law compels platforms to be more transparent by, for inst
...
 The French Superior Council of the Audiovisual (CSA) has been given the authority to hinder or interrupt the broadcasting of foreign state-owned (or state-influenced) TV-services in cases where fundamental national interests are at stake.
 
+#### Jurisprudence
+
+On May 10, 2019 two French politicians (a European deputy and a senator) have asked for the **takedown of a Tweet** from the Minister of Interior claiming that a hospital in Paris was attacked during a demonstration on May 1 by a group Yellow Vests. However, it has been proven that these people were seeking refuge and had no intent to harm the hospital whatsoever.
+
+Based on the 2018 law against information manipulation, they went to court on May 16 but  the **decision to clear both the Minister and Twitter** was given on May 17. 
+
+It was judged that said law was to restrictive to comprehend the case within its scope, as **not [all criteria](https://www.legifrance.gouv.fr/affichCodeArticle.do?cidTexte=LEGITEXT000006070239&idArticle=LEGIARTI000037850595&dateTexte=&categorieLien=cid) were met**.
```
Comment (MattiSG) [inline _pages/encyclopedia/4-reaction/1-case-studies.md]: “too restrictive” is a judgement, and it is not what was ruled. I'm not sure I understand your point here. The law was applied.

File: _pages/encyclopedia/4-reaction/1-case-studies.md [hunk 5]
```diff
@@ -30,6 +28,25 @@ During said times, the law compels platforms to be more transparent by, for inst
...
+#### Jurisprudence
+
+On May 10, 2019 two French politicians (a European deputy and a senator) have asked for the **takedown of a Tweet** from the Minister of Interior claiming that a hospital in Paris was attacked during a demonstration on May 1 by a group Yellow Vests. However, it has been proven that these people were seeking refuge and had no intent to harm the hospital whatsoever.
+
+Based on the 2018 law against information manipulation, they went to court on May 16 but  the **decision to clear both the Minister and Twitter** was given on May 17. 
+
+It was judged that said law was to restrictive to comprehend the case within its scope, as **not [all criteria](https://www.legifrance.gouv.fr/affichCodeArticle.do?cidTexte=LEGITEXT000006070239&idArticle=LEGIARTI000037850595&dateTexte=&categorieLien=cid) were met**. 
+
+The content should be:
+- **false or misleading allegations or attributions** - however the content of the Tweet relied on actual events (an intrusion in the hospital) and deemed only an exaggeration;
```
Comment (MattiSG) [inline _pages/encyclopedia/4-reaction/1-case-studies.md]: Use quotes here!!

File: _pages/encyclopedia/4-reaction/1-case-studies.md [hunk 6]
```diff
@@ -30,6 +28,25 @@ During said times, the law compels platforms to be more transparent by, for inst
...
+- of a nature **able to alter the sincerity of the next election** - but the strong negative reactions to the Tweet were said to be enough to allow citizens to build their own enlightend opinion without being influenced.
```
Comment (MattiSG) [inline _pages/encyclopedia/4-reaction/1-case-studies.md]: No capital to “tweet”.
“their own enlightend opinion without being influenced” → “their own opinion”

File: _pages/encyclopedia/4-reaction/1-case-studies.md [hunk 7]
```diff
@@ -30,6 +28,25 @@ During said times, the law compels platforms to be more transparent by, for inst
...
+
+Based on the 2018 law against information manipulation, they went to court on May 16 but  the **decision to clear both the Minister and Twitter** was given on May 17. 
+
+It was judged that said law was to restrictive to comprehend the case within its scope, as **not [all criteria](https://www.legifrance.gouv.fr/affichCodeArticle.do?cidTexte=LEGITEXT000006070239&idArticle=LEGIARTI000037850595&dateTexte=&categorieLien=cid) were met**. 
+
+The content should be:
+- **false or misleading allegations or attributions** - however the content of the Tweet relied on actual events (an intrusion in the hospital) and deemed only an exaggeration;
+- of a nature **able to alter the sincerity of the next election** - but the strong negative reactions to the Tweet were said to be enough to allow citizens to build their own enlightend opinion without being influenced. 
+Its dissemination should be:
+- **deliberate, artificial or automated** - but no third party was paid to so accentuate the dissemination ;
```
Comment (MattiSG) [inline _pages/encyclopedia/4-reaction/1-case-studies.md]: No, this was simply not demonstrated.

[Source](https://twitter.com/PJanuel/status/1131522202048897024).

File: _pages/encyclopedia/4-reaction/1-case-studies.md [hunk 8]
```diff
@@ -30,6 +28,25 @@ During said times, the law compels platforms to be more transparent by, for inst
...
+- **massive** ;
+- **public through an online communication service**.
```
Comment (MattiSG) [inline _pages/encyclopedia/4-reaction/1-case-studies.md]: Wouldn't these criteria be more properly listed above?

File: _pages/encyclopedia/4-reaction/1-case-studies.md [hunk 9]
```diff
@@ -30,6 +28,25 @@ During said times, the law compels platforms to be more transparent by, for inst
...
+
+Also, Twitter France was acquitted as it claimed that it only monetized the information network of Twitter International, who thus was actually responsible for data use and processing.
```
Comment (MattiSG) [inline _pages/encyclopedia/4-reaction/1-case-studies.md]: “acquitted”? Of what was it accused?

File: _pages/encyclopedia/4-reaction/1-case-studies.md [hunk 10]
```diff
@@ -30,6 +28,25 @@ During said times, the law compels platforms to be more transparent by, for inst
...
+- **false or misleading allegations or attributions** - however the content of the Tweet relied on actual events (an intrusion in the hospital) and deemed only an exaggeration;
+- of a nature **able to alter the sincerity of the next election** - but the strong negative reactions to the Tweet were said to be enough to allow citizens to build their own enlightend opinion without being influenced. 
+Its dissemination should be:
+- **deliberate, artificial or automated** - but no third party was paid to so accentuate the dissemination ;
+- **massive** ;
+- **public through an online communication service**.
+
+Also, Twitter France was acquitted as it claimed that it only monetized the information network of Twitter International, who thus was actually responsible for data use and processing. 
+
+Moreover, the two initiators were ordered to bear the costs of the proceedings.
```
Comment (MattiSG) [inline _pages/encyclopedia/4-reaction/1-case-studies.md]: This is standard practice when the case is lost, I don't see the added value of adding this precision.

File: _pages/encyclopedia/4-reaction/1-case-studies.md (hunk 3 above)
Comment (BerylBogui) [reply _pages/encyclopedia/4-reaction/1-case-studies.md]: Non, c'est bien le bon déroulement :)
- cour saisie le 10
- audience le 16
- décision le 17

File: _pages/encyclopedia/4-reaction/1-case-studies.md (hunk 8 above)
Comment (BerylBogui) [reply _pages/encyclopedia/4-reaction/1-case-studies.md]: Completed

**Comment by BerylBogui** [comment]
Updated according to Matti's review

File: _pages/encyclopedia/4-reaction/1-case-studies.md (hunk 3 above)
Comment (MattiSG) [reply _pages/encyclopedia/4-reaction/1-case-studies.md]: Ok mais il faut documenter le délai de 6 jours entre la saisie et l'audience ! Dans l'idée c'est 48h…

File: _pages/encyclopedia/4-reaction/1-case-studies.md [hunk 11]
```diff
@@ -17,19 +17,34 @@ permalink: /encyclopedia/reaction/case-studies
 
 ### The French law against information manipulation
 
-In France, a law against information manipulation and the **intentional** spread of disinformation was officially adopted on November 20, 2018, nine months after its proposal.
-
-<!-- add decree publication date -->
+In France, a law against information manipulation and the **intentional** spread of disinformation was officially adopted on November 20, 2018, nine months after its proposal and enacted on December 22, 2018.
```
Comment (MattiSG) [inline _pages/encyclopedia/4-reaction/1-case-studies.md]: Non ! Les décrets [ont été publiés](https://www.legifrance.gouv.fr/affichLoiPubliee.do?idDocument=JORFDOLE000037151987&type=echeancier&typeLoi=&legislature=15) le 10 avril.

File: _pages/encyclopedia/4-reaction/1-case-studies.md [hunk 12]
```diff
@@ -17,19 +17,34 @@ permalink: /encyclopedia/reaction/case-studies
...
 
 It focuses on massive and rapid spreads of disinformation on social media and more widely digital tools, including foreign state-owned media outlets, and was meant to be especially efficient in electoral contexts.
 
 During said times, the law compels platforms to be more transparent by, for instance, reporting sponsored content as well as the name and invested resources of sponsors. The biggest, most used platforms are to have a legal representative in France and publish their algorithms as open data.
 
-"Fake news" are also formally defined in order to allow for judiciary action. This definition includes massive and artificial diffusion, capacity of disturbance (of public peace or elections), and undeniability of the falsehood of the information being spread.
+"Fake news" are also formally defined in order to allow for judiciary action. This definition includes deliberate, massive and artificial public diffusion, capacity of disturbance (of public peace or elections), and undeniability of the falsehood or misleading nature of the information being spread.
```
Comment (MattiSG) [inline _pages/encyclopedia/4-reaction/1-case-studies.md]: “capacity for” ou plutôt “capability of”, “capacity of” signifie “capacité” dans le sens de la contenance.

```suggestion
"Fake news" are also formally defined in order to allow for judiciary action. This definition includes deliberate, massive and artificial public diffusion, capability for disruption (of public peace, elections…), and undeniability of the falsehood or misleading nature of the information being spread.
```

File: _pages/encyclopedia/4-reaction/1-case-studies.md [hunk 13]
```diff
@@ -17,19 +17,34 @@ permalink: /encyclopedia/reaction/case-studies
...
 
 Outside of election times, platforms ought to cooperate, as well as build and implement open measures to fight disinformation.
 The French Superior Council of the Audiovisual (CSA) has been given the authority to hinder or interrupt the broadcasting of foreign state-owned (or state-influenced) TV-services in cases where fundamental national interests are at stake.
 
+#### [Jurisprudence](https://www.dalloz-actualite.fr/flash/loi-fake-news-premiere-application-du-refere)
+
+On May 10, 2019 two French elected representatives have asked for the takedown of a [tweet](https://twitter.com/ccastaner/status/1123664392011304961) from the Minister of Interior claiming that a hospital in Paris was attacked during a demonstration on May 1 by a group Yellow Vests. However, it has been proven that these people were [only seeking refuge](https://www.lemonde.fr/politique/article/2019/05/02/incidents-a-la-pitie-salpetriere-castaner-contredit-par-une-video-l-opposition-demande-des-explications_5457544_823448.html).
```
Comment (MattiSG) [inline _pages/encyclopedia/4-reaction/1-case-studies.md]:
```suggestion
On May 10, 2019 two French elected representatives asked for the takedown of a [tweet](https://twitter.com/ccastaner/status/1123664392011304961) by the minister of Interior claiming that a hospital in Paris had been attacked during a demonstration on May 1. However, it was shortly after [proven](https://www.lemonde.fr/politique/article/2019/05/02/incidents-a-la-pitie-salpetriere-castaner-contredit-par-une-video-l-opposition-demande-des-explications_5457544_823448.html) that these people were only seeking refuge.
```

File: _pages/encyclopedia/4-reaction/1-case-studies.md [hunk 14]
```diff
@@ -17,19 +17,34 @@ permalink: /encyclopedia/reaction/case-studies
...
+
+Based on the 2018 law against information manipulation, they brought the case to court on May 10 and obtained a hearing on the 16th, but  the decision to clear both the Minister and Twitter was given on May 17.
```
Comment (MattiSG) [inline _pages/encyclopedia/4-reaction/1-case-studies.md]: From what I understand Twitter was not liable, and thus did not have to be “cleared” of anything. The ruling could have resulted in an obligation for Twitter to delete that tweet, but it did not.

File: _pages/encyclopedia/4-reaction/1-case-studies.md [hunk 15]
```diff
@@ -17,19 +17,34 @@ permalink: /encyclopedia/reaction/case-studies
...
 
+#### [Jurisprudence](https://www.dalloz-actualite.fr/flash/loi-fake-news-premiere-application-du-refere)
+
+On May 10, 2019 two French elected representatives have asked for the takedown of a [tweet](https://twitter.com/ccastaner/status/1123664392011304961) from the Minister of Interior claiming that a hospital in Paris was attacked during a demonstration on May 1 by a group Yellow Vests. However, it has been proven that these people were [only seeking refuge](https://www.lemonde.fr/politique/article/2019/05/02/incidents-a-la-pitie-salpetriere-castaner-contredit-par-une-video-l-opposition-demande-des-explications_5457544_823448.html).
+
+Based on the 2018 law against information manipulation, they brought the case to court on May 10 and obtained a hearing on the 16th, but  the decision to clear both the Minister and Twitter was given on May 17. 
+
+According to the judgement, the law does not comprehend the case within its scope, as **not [all criteria](https://www.legifrance.gouv.fr/affichCodeArticle.do?cidTexte=LEGITEXT000006070239&idArticle=LEGIARTI000037850595&dateTexte=&categorieLien=cid) were met**.
```
Comment (MattiSG) [inline _pages/encyclopedia/4-reaction/1-case-studies.md]:
```suggestion
According to the ruling, the case was out of the scope of the law, as **not [all criteria](https://www.legifrance.gouv.fr/affichCodeArticle.do?cidTexte=LEGITEXT000006070239&idArticle=LEGIARTI000037850595&dateTexte=&categorieLien=cid) were met**. 
```

File: _pages/encyclopedia/4-reaction/1-case-studies.md [hunk 16]
```diff
@@ -17,19 +17,34 @@ permalink: /encyclopedia/reaction/case-studies
...
+
+Based on the 2018 law against information manipulation, they brought the case to court on May 10 and obtained a hearing on the 16th, but  the decision to clear both the Minister and Twitter was given on May 17. 
+
+According to the judgement, the law does not comprehend the case within its scope, as **not [all criteria](https://www.legifrance.gouv.fr/affichCodeArticle.do?cidTexte=LEGITEXT000006070239&idArticle=LEGIARTI000037850595&dateTexte=&categorieLien=cid) were met**. 
+
+The content should be:
+- **"false or misleading allegations or attributions"** - however the content of the tweet relied on actual events (an intrusion in the hospital) and deemed only an exaggeration;
+- of a nature **"able to alter the sincerity of the next election"** - but the strong negative reactions to the Tweet were said to be enough to allow citizens to build their own opinion.
```
Comment (MattiSG) [inline _pages/encyclopedia/4-reaction/1-case-studies.md]:
```suggestion
- of a nature **"able to alter the sincerity of the next election"** - but the strong negative reactions to the tweet were said to be enough to allow citizens to build their own opinion. 
```

File: _pages/encyclopedia/4-reaction/1-case-studies.md [hunk 17]
```diff
@@ -17,19 +17,34 @@ permalink: /encyclopedia/reaction/case-studies
...
+
+The content should be:
+- **"false or misleading allegations or attributions"** - however the content of the tweet relied on actual events (an intrusion in the hospital) and deemed only an exaggeration;
+- of a nature **"able to alter the sincerity of the next election"** - but the strong negative reactions to the Tweet were said to be enough to allow citizens to build their own opinion. 
+Its dissemination should be:
+- **"deliberate, artificial or automated"** - but this was not demonstrated ;
+- **"massive"** ;
+- **"public through an online communication service"**.
+
+Also, Twitter France was not held responsible as it claimed that it only monetized the information network of Twitter International, who thus was actually responsible for data use and processing.
```
Comment (MattiSG) [inline _pages/encyclopedia/4-reaction/1-case-studies.md]: I am not sure it was “not held responsible”, but rather that it argued (and won) that it should not be cited at all, but rather Twitter International.

**Pull request merged**